### DISCUSSION TITLE
Somehow limit the amount of eco-ci boxes

### DISCUSSION THREAD

**Discussion by ribalba** [green-coding-solutions/green-metrics-tool#466] [Ideas]
If you look at a PR like:

https://github.com/green-coding-berlin/green-metrics-tool/pull/430

it becomes quite hard to read as after every commit you see the eco-ci graph/ box. Is there a way we can minimise these? Or only show the last run as in tests? No real idea how to improve this but it is quite annoying when trying to understand what people have done in PRs

**Comment by ArneTR** [comment]
You can hide the comment if you want. This is some manual work, but I think it will stay. What do you think about that?

**Comment by ribalba** [comment]
can we do this automatically when a new run is posted? Otherwise it is quite annoying to go through all the comments on PRs

<img width="954" alt="Screenshot 2023-09-19 at 09 24 36" src="https://github.com/green-coding-berlin/green-metrics-tool/assets/39629/d0527693-63a5-404c-a89f-277b7a36f479">

**Reply by ribalba** [reply]
Maybe: https://docs.github.com/en/graphql/reference/mutations#minimizecomment

**Reply by dan-mm** [reply]
This is a good call, I'll see what's possible here. I think ideally it would on posting a new measurement, minimize the old ones. I think that'd be the best user experience

**Reply by ArneTR** [reply]
+1 on Dans recommendation

In the end I think the best would be not only having a constant stream of updates, but also the difference to the measurement before.

But I have that backlogged already
